tests/lib/rules/sort-imports.js

```diff
@@ -287,6 +297,15 @@ ruleTester.run("sort-imports", rule, {
                 data: { memberName: "qux" },
                 type: "ImportSpecifier"
             }]
+        },
+
+        // https://github.com/eslint/eslint/issues/12951
+        {
+            code:
+                "import b from 'foo.js';\n\n" +
+                "import a from 'bar.js';",
+            output: null,
+            errors: [expectedError]
```

Comment:
It would be nice to have an `invalid` test where the option is explicitly set to its default value.

Can you please also add some tests with multiline declarations and some tests where both declarations are on the same line? This is to ensure that the code that counts lines and checks the result works well.

Also, tests with a comment between, and tests with a statement between instead of blank lines.